src/components/gig-detail.html

```diff
@@ -53,44 +53,46 @@
 
   </style>
 
-  <div class="gig-detail">
-    <header class="gig-header">
-      <h1>[[gig.title]]</h1>
-
-      <div class="meta">
-        <div class="location">
-          <iron-icon icon="polygigs:place"></iron-icon> [[gig.location]]
-          <template is="dom-if" if="[[gig.isRemote]]">
-            <div class="badge">Allows Remote</div>
-          </template>
+  <template is="dom-if" if="[[gig]]">
```

Comment:
There's a performance benefit. By putting it in a `template` it is inert and the browser parser totally ignores it. Lets you define HTML but defer its interpretation.